vim/vscode-neovim.vim

```diff
@@ -23,24 +23,22 @@ function! VSCodeNotify(cmd, ...)
     call rpcnotify(g:vscode_channel, s:vscodeCommandEventName, a:cmd, a:000)
 endfunction
 
-function! VSCodeExtensionCall(cmd, ...) abort
-    call rpcrequest(g:vscode_channel, s:vscodePluginEventName, a:cmd, a:000)
-endfunction
-
 function! VSCodeExtensionNotify(cmd, ...)
     call rpcnotify(g:vscode_channel, s:vscodePluginEventName, a:cmd, a:000)
 endfunction
 
 function! VSCodeCallRange(cmd, line1, line2, leaveSelection, ...) abort
-    call VSCodeExtensionCall('range-command', a:cmd, 'V', a:line1, a:line2, 1, 1, a:leaveSelection, a:000)
+    " FIXME: breaking
```

Comment:
fixme

Reply:
In theory, this is a breaking change, but in practice, it has no impact. It needs to be addressed and improved in the PR for redesigning the API

Reply:
We will need to maintain the vimscript API because many people's configs rely on it. I agree the internal implementation can be redesigned though.

Reply:
Yes, the old VimScript API is worth keeping. Mainly the internal implementation and Lua API need to be rewritten.